inference/app/inference.py

```diff
@@ -34,6 +41,32 @@ def allowed_filesize(filesize):
         return False
 
 
+class Models(object):
+    """Models"""
+
+    def __init__(self, model_id):
+        self.model_id = model_id
+        self.model_type = ['docker', 'binary', 's3']
+        self.current_data_state = ''
+        self.model_state = ''
+
+    def execute(self, current_data_state, model_state):
+        pass
+
+    def get_id(self):
+        return self.model_id
+
+
+class Registry(object):
+    """Registry"""
+
+    def __init__(self):
+        self.models_registry = ['vizml', 'vizrec', 'data2vis', 'draco']
```

Comment:
model_registry would be `docker` `git` etc.
VizML, Data2Vis etc. are models that would be in the registry.